unabridged-works-volume-2/original/06-book-common-prayer.adoc

```diff
@@ -384,11 +384,12 @@ which, if he could singly apply himself to wait on the Lord in the meek, sober s
 out of the consultations, wisdom, and confidence of the flesh,
 it might please the Lord to make manifest to him.
 
+[.offset]
```

Comment:
So there are a series of offsets here because there are paragraphs below them that are answers. I wasn't sure if this was the right way to do it, but it seemed like they should be separated from the other text, but his 2nd and 3rd points do not have an answer below them, and they are lumped together. So idk!!

Reply:
yeah, hmmm... I think, upon reflection, that I'd prefer to just remove these offsets and let the text run together. If they were all clearly delineated (not skipping the 2nd and 3rd), then maybe I would lean the other way.  I think let's have you remove these offets from this section 👍